app/assets/javascripts/chf_ga_events.js

```diff
@@ -2,13 +2,15 @@
 // https://github.com/samvera/hyrax/blob/1e504c200fd9c39120f514ac33cd42cd843de9fa/app/assets/javascripts/hyrax/ga_events.js
 //
 // We add our own more flexible.
+// Can be used by a container or directly by a target; just requires the 'ga-event-data' class on the element holding the data.
 //
 // https://developers.google.com/analytics/devguides/collection/gajs/eventTrackerGuide
 
-$(document).on('click', '*[data-analytics-event]', function(e) {
+$(document).on('click', '.ga-event-data', function(e) {
   _gaq.push(['_trackEvent',
```

Comment:
Yes, I tried to make it work that way but because we were putting the data attributes on the containing div instead of on the image / target elements it wasn't picking them up. I tried a couple of things to preserve use of data attributes this way, but I couldn't get it to work. The only alternatives I saw were:
* replicate the data elements into 3 branches of a conditional. (yuck)
* keep the separate call to `_gaq.push` in chf_image_viewer.js. I preferred to move this out and have all the ga events handled by this js file, having earlier missed the separate call entirely because it didn't use this convention.